src/ServiceResetError.c

```diff
@@ -84,7 +84,13 @@ void Ros_ServiceResetError_Trigger(const void* request_msg, void* response_msg)
     }
 
     // Check for condition that can be fixed remotely
-    if (Ros_Controller_IsAlarm())
+    if (Ros_Controller_IsMajorAlarm())
+    {
+        rosidl_runtime_c__String__assign(&response->message, "Major alarm active. Cannot be reset. Check teach pendant");
```

Comment:
Ideally we'd use ` motoros2_interfaces__msg__MotionReadyEnum__NOT_READY_MAJOR_ALARM_STR` instead of a custom string. But seeing the rest of the function, perhaps that would be out-of-scope for this PR.

Reply:
Yeah I would prefer to use `motoros2_interfaces__msg__MotionReadyEnum__NOT_READY_MAJOR_ALARM_STR` but I decided to keep it consistent with the rest of the function. I can make another PR that changes the rest of the function to match in the future.